integration-cli/docker_api_containers_test.go

```diff
@@ -1948,3 +1949,27 @@ func (s *DockerSuite) TestContainerKillCustomStopSignal(c *check.C) {
 	err = waitInspect(id, "{{.State.Running}} {{.State.Restarting}}", "false false", 30*time.Second)
 	c.Assert(err, checker.IsNil)
 }
+
+func (s *DockerSuite) TestCpAPIWithHardlink(c *check.C) {
```

Comment:
The API is called `archive` not `Co`, so I think this test should be called:  `TestContainerAPIArchiveWithHardlink`.

It looks like a bunch of the tests in this files aren't following that pattern, but I believe that's the original pattern.